linux-gcc-5-qt-5/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM mbgl/ci:latest-linux-gcc-5
+
+# --------------------------------------------------------------------------------------------------
+# Set environment variables
+
+WORKDIR /opt
+
+RUN set -eu \
+ && sed -i 's/# deb-src/deb-src/' /etc/apt/sources.list \
+ && apt-get update \
+ && apt-get build-dep -y qt5-default \
+ && apt-get install -y libc6-dbg libxinerama-dev libxcb-xinerama0-dev
+
+RUN set -eu \
+ && git clone git://code.qt.io/qt/qt5.git
+
+WORKDIR /opt/qt5
+
+RUN set -eu \
+ && git checkout 5.9
+
+RUN set -eu \
+ && ./init-repository --branch \
+    --module-subset=qtbase,qtxmlpatterns,qtdeclarative,qtgraphicaleffects,qtquickcontrols,qtsvg,qtimageformats,qtlocation,qtserialport,qttools
+
+WORKDIR /opt/qt5/build
+
+RUN set -eu \
+ && mkdir -p /opt/qt-5.9 \
+ && ../configure \
+    -opensource -confirm-license \
+    -nomake tests -nomake examples \
+    -debug -prefix /opt/qt-5.9 \
```

Comment:
It turns out having a debug Qt5 build had big performance implications when running memory load tests under valgrind - I was obtaining CI runs up to 23 minutes long. Switching the Qt5 build to release decreased the docker image size from 3G to 1.5G (uncompressed) and also made it possible to switch the amount of jobs back to 6 (used by other jobs as default).